source/base/image/openexr.cpp

```diff
@@ -105,17 +95,17 @@ class POV_EXR_OStream : public Imf::OStream
 				throw POV_EXCEPTION(kFileDataErr, "Error while writing EXR output");
 		}
 
-		Int64 tellp()
+		unsigned long tellp()
 		{
 			unsigned long pos = os.tellg();
 			if((int) pos == -1)
```

Comment:
While you're at it, all `unsigned long` in the file should be considered a bug. They should either be `POV_LONG` (v3.7) / `POV_OFF_T` (v3.8) or whatever we'll be using instead of `Int64`.

Same goes for the comparison with -1 here and further below: There's a bug lurking in there as well, and it should instead be something like `if(pos == POV_LONG(-1))` (or correspondingly for v3.8).